### PR TITLE
Polish locale

### PR DESCRIPTION
**Pull request by filiptepper**
Polish locale for forem, coming right up!

### CODE REVIEW

File: config/locales/pl.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+en:
```
Comment (parndt) [inline config/locales/pl.yml]: this should be `pl` not `en` :D

Comment (filiptepper) [inline config/locales/pl.yml]: Geez… ;-)

Comment (filiptepper) [inline config/locales/pl.yml]: Fixed!

**Comment by parndt** [comment]
Thanks @filiptepper !

**Pull request merged**